payu/models/staged_cable.py

```diff
@@ -47,30 +30,45 @@ def __init__(self, expt, name, config):
         self.model_type = 'staged_cable'
         self.default_exec = 'cable'
 
+        # We want people to be able to use payu during testing, which
+        # often means additions of new namelists due to new science
+        # modules. I would like to set
+        # optional_config_files = glob.glob("*.nml")
+        # but this feels like a bit of an abuse of feature.
```

Comment:
In this case I don't think this would be a problem. It just creates a list of files that will be copied into the ephemeral `work` directory and tracked with `git`.